config/config.json

```diff
@@ -1,180 +1,387 @@
-      "trimming":"CookieDataDelegate",
+  "defaultProfile": "presenter_presentation",
+  "data": {
+    "enabled": true,
+    "dataDelegates": {
+      "trimming": "CookieDataDelegate",
       "visualAnnotations": "VisualAnnotationsDataDelegate",
-      "metadata":"VideoManifestMetadataDataDelegate",
-      "cameraTrack":"TrackCameraDataDelegate"
+      "metadata": "VideoManifestMetadataDataDelegate",
+      "cameraTrack": "TrackCameraDataDelegate"
     }
   },
   "folders": {
     "profiles": "config/profiles",
     "resources": "resources",
     "skins": "resources/style"
   },
-  "experimental":{
-    "autoplay":true
+  "experimental": {
+    "autoplay": true
   },
-  "plugins":{
+  "plugins": {
     "enablePluginsByDefault": false,
-
-    "//**** Instructions: Disable any individual plugin by setting its enable property to false": {"enabled": false},
-    "//**** For a list of available plugins and configuration, go to": "https://github.com/polimediaupv/paella/blob/master/doc/plugins.md",
-    "list":{
-      "//******* Button plugins": "",
-      "edu.harvard.dce.paella.flexSkipPlugin": {"enabled":true, "direction": "Rewind", "seconds": 10},
-      "edu.harvard.dce.paella.flexSkipForwardPlugin": {"enabled":true, "direction": "Forward", "seconds": 30},
-      "es.upv.paella.captionsPlugin": {"enabled":true, "searchOnCaptions":true},
-      "es.upv.paella.extendedTabAdapterPlugin": {"enabled":true},
-      "es.upv.paella.footprintsPlugin": {"enabled":false},
-      "es.upv.paella.frameControlPlugin":  {"enabled": true, "showFullPreview": "auto", "showCaptions":true},
-      "es.upv.paella.fullScreenButtonPlugin": {"enabled":true, "reloadOnFullscreen":{ "enabled":true, "keepUserSelection":true }},
-      "es.upv.paella.helpPlugin": {"enabled":true, "langs":["en","es"]},
-      "es.upv.paella.multipleQualitiesPlugin": {"enabled":true, "showWidthRes":true},
-      "es.upv.paella.playbackRatePlugin": {"enabled":true, "availableRates": [0.75, 1, 1.25, 1.5]},
-      "es.upv.paella.playPauseButtonPlugin": {"enabled":true},
-      "es.upv.paella.searchPlugin": {"enabled":true, "sortType":"time", "colorSearch":false},
-      "es.upv.paella.socialPlugin": {"enabled":true},
-      "es.upv.paella.themeChooserPlugin":  {"enabled":true},
-      "es.upv.paella.viewModePlugin": { "enabled": true },
-      "es.upv.paella.volumeRangePlugin":{"enabled":true, "showMasterVolume": true, "showSlaveVolume": false },
-      "es.upv.paella.pipModePlugin": { "enabled":true },
-      "es.upv.paella.ratePlugin": { "enabled":true },
-      "es.upv.paella.videoZoomPlugin": { "enabled":true },
-      "es.upv.paella.audioSelector": { "enabled":true },
-      "es.upv.paella.videoZoomToolbarPlugin": { "enabled":false, "targetStreamIndex":0 },
-      "es.upv.paella.videoZoomTrack4kPlugin": { "enabled":true, "targetStreamIndex":0, "autoModeByDefault":false },
-      "es.upv.paella.airPlayPlugin": { "enabled":true },
-
-      "//***** Video Overlay Button Plugins": "",
```

Comment:
What is the value of this in a configuration file?

Reply:
@staubesv  I find these short, releavant comments, that are close to the data to be extremenly useful guides. The same way the short relevant comments in the Opencast properties file and code are give a boost of context and design motivation.  The alternative is to require users to parse through a local README or compile a remote set of markup docs. It's most likely, an supported by your "obvious" comment,  that people will struggle with what they consider obvious based on property names, and only read the instructions as a last resort. The inline comments gives greater accessibility to the obvious.

Reply:
@karendolan Can you provide an example of a relevant comment? At least for me, it doesn't help me anything to know that a plugin that I can enable is of a specific type (e.g. Video Overlay Button Plugin). To know, for example, what enabling es.upv.paella.videoZoomTrack4kPlugin will cause I need a documentation of that plugin. 
Since JSON does not support comments, it seems the wrong place to add such information as description etc.

Anyway... no strong opinion about the comments. In first place, this PR is about having a properly formatted JSON.
Note though that automatic formatters will not deliver good results for data that actually is supposed to be "comments" as this would require a different formatting for optimum results.